pkg/wipeout/wipeout.go

```diff
@@ -0,0 +1,201 @@
+// Copyright 2021 Google LLC. All Rights Reserved.
```

Comment:
This is the first file in `pkg`, which is a nod to #593

Reply:
Perhaps we should soon do at least the `pkg` part of that refactor to avoid divergence.

Reply:
+1 let's discuss further in #593